scipy/sparse/linalg/dsolve/_superluobject.c

```diff
@@ -890,7 +890,7 @@ static int rowperm_cvt(PyObject * input, rowperm_t * value)
 {
     ENUM_CHECK_INIT;
     ENUM_CHECK(NOROWPERM);
-    ENUM_CHECK(LargeDiag);
+    ENUM_CHECK(LargeDiag_MC64);
```

Comment:
This should probably be removed since MC64 is not included.

Reply:
Agree, it's not used in SciPy codebase and tests pass without it.